### PR TITLE
Updated GWAS pre-processing step to remove HASGs from BioMart

### PR DESCRIPTION
**Pull request by jaclynbeck-sage**
This PR updates the list of igap genes by removing human alternative sequence Ensembl IDs from the list. These IDs ended up in the original list because of the BioMart query to get Ensembl IDs, not because they were provided by the original data source, so it is safe to remove them. The igap list now contains only Ensembl IDs for genes on numeric chromosomes or X, Y, or MT chromosomes, with all HASGs and patches removed. 

Since diffs of notebooks are hard to parse, I've commented where the two major changes are. The human-readable notebook is here: https://github.com/Sage-Bionetworks/agora-data-tools/blob/jbeck/AG-1250/filter_HASGs_igap/data_analysis/agora/notebooks/preprocessing/AG-796_process_gwas.ipynb

### CODE REVIEW

File: data_analysis/agora/notebooks/preprocessing/AG-796_process_gwas.ipynb [hunk 1]
```diff
@@ -305,7 +317,7 @@
     }
    ],
    "source": [
-    "attributes = ['ensembl_gene_id', 'external_gene_name']\n",
+    "attributes = ['ensembl_gene_id', 'external_gene_name', 'chromosome_name']\n",
```
Comment (jaclynbeck-sage) [inline data_analysis/agora/notebooks/preprocessing/AG-796_process_gwas.ipynb:320]: Relevant change: Added chromosome name to the query so we can filter out HASGs.

File: data_analysis/agora/notebooks/preprocessing/AG-796_process_gwas.ipynb [hunk 2]
```diff
@@ -323,10 +335,55 @@
...
+    "    # Keep rows if they have a numerical chromosome name, or have X, Y, or MT\n",
+    "    lambda row: re.match(regex, row) is not None if type(row) == str else True # Always true for numbers \n",
+    ")\n",
+    "print(keep.value_counts())\n",
+    "\n",
+    "result = result.loc[keep].reset_index(drop = True)\n",
+    "result = result[['ensembl_gene_id', 'hgnc_symbol']]\n",
+    "len(result)"
+   ]
+  },
```
Comment (jaclynbeck-sage) [inline data_analysis/agora/notebooks/preprocessing/AG-796_process_gwas.ipynb:386]: This section is new and does the filtering based on a regex that recognizes numerical chromosomes or X, Y, and MT.

**Review by JessterB — APPROVED** [review]
lgtm!
